auth/console/policy_make_command_test.go

```diff
@@ -20,5 +20,11 @@ func TestEventMakeCommand(t *testing.T) {
 	err = policyMakeCommand.Handle(mockContext)
 	assert.Nil(t, err)
 	assert.True(t, file.Exists("app/policies/user_policy.go"))
+
+	mockContext.On("Argument", 0).Return("User/AuthPolicy").Once()
+	err = policyMakeCommand.Handle(mockContext)
+	assert.Nil(t, err)
+	assert.True(t, file.Exists("app/policies/User/auth_policy.go"))
```

Comment:
S: Add assert: `assert.True(file.Contain("app/policies/User/auth_policy.go", "package User"))` and `assert.True(file.Contain("app/policies/User/auth_policy.go", "type AuthPolicy struct {"))`